src/ucs/sys/compiler_def.h

```diff
@@ -106,6 +106,12 @@
         ( sizeof(_array) / sizeof((_array)[0]) ); \
     })
 
+/**
+ * @return count of elements in const-size array
+ */
+#define ucs_countof(_array) \
```

Comment:
isn't this duplicating ucs_static_array_size() above?

Reply:
ucs_static_array_size has more strict syntax - it require static variable as argument, else - it produces compilation error, ucs_countof macro is less strict but may allow logical errors

Reply:
so call it ucs_array_size()?

Reply:
renamed